spec/rspec/core/configuration_options_spec.rb

```diff
@@ -296,16 +296,16 @@
   end
 
   describe "--fail-fast" do
-    it "defaults to false" do
-      expect(parse_options[:fail_fast]).to be_falsey
+    it "defaults to nil" do
```

Comment:
These concern me as they represent a change in how we document them, if they are wrong I almost want to fix them rather than the specs.

Reply:
I didn't check the implementation, but I find it perfect that the setting is `nil` if there's no CLI option, and `1` or `false` if there's an explicit CLI option. For example it makes it possible to override a `nil` setting with a local setting file, but `false` should not be overriden.
Or are you talking about `1` vs `true`?